accord-core/src/main/java/accord/local/CommandStores.java

```diff
@@ -358,7 +358,7 @@ private synchronized TopologyUpdate updateTopology(Node node, Snapshot prev, Top
                 bootstrapUpdates.add(shard.store.interruptBootstraps(epoch, newRanges.currentRanges()));
             }
             // TODO (desired): only sync affected shards
-            if (epoch > 1)
+            if (epoch > 1 && startSync)
```

Comment:
rewritten in https://issues.apache.org/jira/browse/CASSANDRA-18519, the logic is now

```
Map<Boolean, Ranges> partitioned = add.partitioningBy(range -> shouldBootstrap(node, prev.local, newLocalTopology, range));
if (partitioned.containsKey(true))
    bootstrapUpdates.add(shardHolder.store.bootstrapper(node, partitioned.get(true), newLocalTopology.epoch()));
if (partitioned.containsKey(false))
    bootstrapUpdates.add(() -> shardHolder.store.initialise(epoch, partitioned.get(false)));
```

Reply:
I feel the `startSync` is to work around the bug where we don't want to bootstrap when a keyspace is added?  If so I added a new `accord.local.CommandStores#shouldBootstrap` for that... 

One issue to deal with is that we could have a new keyspace at the same time we see a range movement; so we need to bootstrap the range movement but not the new keyspace